app/assets/javascripts/discourse/components/topic-list-item.js

```diff
@@ -67,6 +69,16 @@ export default Component.extend({
         }
       });
     }
+
+    schedule("afterRender", () => {
+      const rawTopicLink = this.element.querySelector(".raw-topic-link");
```

Comment:
Do we need a check here in case `this.element` is `null` say on destroy?

Reply:
yes I was thinking of it, at the minimum I think it's safer will add a guard :+1: